ts/components/SectionStatusComponent.tsx

```diff
@@ -67,19 +70,38 @@ const SectionStatusComponent: React.FC<Props> = (props: Props) => {
     return null;
   }
 
+  const viewRef = React.createRef<View>();
   const sectionStatus = props.sectionStatus;
   const iconName = statusIconMap[sectionStatus.level];
   const backgroundColor = statusColorMap[sectionStatus.level];
   const locale = getFullLocale();
   const maybeWebUrl = maybeNotNullyString(
     sectionStatus.web_url && sectionStatus.web_url[locale]
   );
+
+  const handleOnSectionRef = () => {
+    if (viewRef.current) {
+      props.onSectionRef?.(viewRef);
+    }
+  };
+
+  React.useEffect(() => {
+    handleOnSectionRef();
+
+    const unsubscribe = props.navigationProps?.addListener(
```

Comment:
As a refinement you could avoid to pass the navigations props
`navigationProps?: NavigationScreenProp<NavigationState>;`
by using the equivalent hook
```typescript
const navigation = useNavigationContext();
[...]
const unsubscribe = navigation.addListener("didFocus", handleOnSectionRef);
```